src/components/Header.jsx

```diff
@@ -138,7 +138,7 @@ const Header = ({
         ) : (
           <>
             <div className={`${elementStyles.info} mr-3`}>
-              Logged in as @{userId}
+              Logged in as {userId ? `@${userId}` : email}
```

Comment:
this could also be shifted into `useLoginContext` (perhaps as `displayName`?) along with exposing an extra `isLoggedIn` prop that's like so `isLoggedIn: userId || email`. 

small aside - could we double check w/ designers if we can have both as `@<type>`

Reply:
Checked with design - we're avoiding `@email` because it becomes confusing for users (`@blah@blah.com`). Have swapped to a generic `accountName` to determine whether a user is logged in - not adding a util function just for the display name since it only appears in 2 places though